schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
+					"items": {
+						"type": "string"
```

Add codelist keywords.

```suggestion
						"type": "string",
						"codelist": "country.csv",
						"openCodelist": false
```